desugar/builtins.py

```diff
@@ -108,13 +111,14 @@ def __getattribute__(self, attr: str, /) -> Any:
 
         if attr in self.__dict__:
             return self.__dict__[attr]
+        elif descriptor_type_get is not NOTHING:
```

Comment:
Why this change?

Reply:
Because this makes the code simpler by avoiding unnecessary nested if statements (as checking `type_attr is not NOTHING` is not necessary as it is already implied by `descriptor_type_get is not NOTHING`).